Automattic/jetpack
# Admin page: display blocks info even when classic editor is active

#### Changes proposed in this Pull Request:

* One can use the block editor even when the classic editor is active. Let's take that into account and not limit the display when that plugin is active.
* See https://github.com/Automattic/jetpack/pull/23766#issuecomment-1086346418

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Install the classic editor plugin on your site.
* Go to Jetpack > Dashboard, and Jetpack > Settings > Writing
* You should still see the blocks cards.
